app/content/utils/mdx.ts

```diff
@@ -26,9 +26,23 @@ const DATASET_CONTENT_PATH = path.join(
 
 const md = markdownit();
 
+function getParentDataset(data) {
+  return {
+    id: data.id,
+  };
+}
+
 function parseAttributes(obj) {
   const convert = (obj) => {
-    return Object.keys(obj).reduce(
+    const mdxData = {
+      ...obj,
+      layers: obj.layers?.map((l) => ({
+        ...l,
+        parentDataset: getParentDataset(obj),
```

Comment:
Could this just be 
```suggestion
        parentDataset:  { id: obj.id },
```
instead of writing a separate function for it? Am I missing something the getParentDataset function additionally does? 🤔

Reply:
Yah I thought using a separate function might make it clear what it is trying to do (for people who don't touch veda much)? But I think we can go either way I incorporated your suggestion since I also had to get rid of the function.